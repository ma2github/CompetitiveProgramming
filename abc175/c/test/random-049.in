184597521652792 172544354698946 678376206672988
