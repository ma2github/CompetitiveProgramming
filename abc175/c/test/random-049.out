184597521652792
